index.html

```diff
@@ -1 +1,55 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
```

Comment:
Suggestion: order and separate paragraph copy based on a WHO, WHAT, WHY, and HOW structure for better context

WHO -> _The Email Markup Consortium (EMC) is a community-led group_...

WHAT -> _Senders, recipients, and email clients share the common goals_... 

WHY -> _Recipients should be able to easily access_... 

HOW -> _The EMC collaborates with developers, creation and sending tools_...

Reply:
I didn't break it up exactly as suggested, but it's a step towards that. We can revisit again later.